src/Database/PostUpdate.php

```diff
@@ -375,4 +376,49 @@ private static function update1281()
 
 		return false;
 	}
+
+	// Post-update script of PR 5596
+	function fixGenderStrings() {
+		$allGenders = DBA::select('contact', ['id', 'gender']);
+		$allLangs = L10n::getAvailableLanguages();
+		$success = 0;
+		$fail = 0;
+		foreach($allGenders as $key=>$gender) {
```

Comment:
Code standards: Please add a space after `foreach` and wrap operators (like `=>`) with spaces.